frontend/app/views/fragments/page/pageHeader.scala.html

```diff
@@ -1,6 +1,6 @@
-@(title: String, introOpt: Option[String] = None)
+@(title: String, introOpt: Option[String] = None, isInline: Boolean = false)
 
-<section class="page-header">
+<section class="page-header@if(!isInline){ page-header--aligned}">
```

Comment:
could this be isBlock? or maybe control this outside of the fragment? what are the use cases?